test/api/api_test.go

```diff
@@ -171,7 +169,11 @@ func RunApiServer(t *testing.T, binary string, storageApiHost string) string {
 
 	var stdout, stderr bytes.Buffer
```

Comment:
BTW `stdout` a `stderr` sa nikde nepouzivaju (?)

Reply:
Aha, ne, to je pozůstatek copy&paste z cli testů, odstraním.

Reply:
A nebo na to přidám aspoň jednoduchý asserty. 🙂

Reply:
Asi staci odstranit.

Reply:
ok 🙂